cli/sparse_dd.ml

```diff
@@ -354,7 +355,7 @@ let _ =
 		t, dest, "raw"
 	| _, _, Some (`Nbd (server, export_name)), _, _, _ ->
 		let dest = rewrite_url dest in
-		let t = Impl.make_stream common (src ^ ":" ^ server ^ ":" ^ export_name) None "nbdhybrid" "raw" in
+		let t = Impl.make_stream common (src ^ ":" ^ server ^ ":" ^ export_name) relative_to "nbdhybrid" "raw" in
```

Comment:
please put back None here, the base_image and the destination_format are not necesarily both Nbd.

Reply:
Thanks, I have added the changes and rebased!